.github/workflows/main.yml

```diff
@@ -123,11 +129,10 @@ jobs:
           -DSDL2IMAGE_SAMPLES=ON \
           -DSDL2IMAGE_TESTS=ON \
           -DSDL2IMAGE_TESTS_INSTALL=ON \
-          -DSDL2IMAGE_AVIF=${{ !matrix.platform.vendored }} \
+          -DSDL2IMAGE_AVIF=ON \
           -DSDL2IMAGE_JXL=ON \
           -DSDL2IMAGE_TIF=ON \
           -DSDL2IMAGE_WEBP=ON \
-          -DSDL2IMAGE_VENDORED=${{ !!matrix.platform.vendored }} \
```

Comment:
It's defaulted to `ON` on MSVC, but I kept it here to be more explicit.
Also, we test vendored libraries on Linux: see the matrix above.

Reply:
How should I change that one? (And feel free to push to my branch.)

Reply:
What would you test? Is only testing Windows sufficient?

Reply:
Because of this, the MSVC build is veeeeeeery slow.

Reply:
Ignore me. Looks like the `-DSDL2IMAGE_VENDORED=` option is duplicated. It's already passed above.